src/backend/marsha/core/tests/test_views_lti_document.py

```diff
@@ -68,6 +68,54 @@ def test_document_view_instructor(self, mock_get_consumer_site, mock_verify):
         # signature)
         self.assertEqual(mock_verify.call_count, 1)
 
+    @mock.patch.object(LTI, "verify")
+    @mock.patch.object(LTI, "get_consumer_site")
+    def test_document_view_administrator(self, mock_get_consumer_site, mock_verify):
+        """Validate the format of the response returned by the view for an admin request."""
```

Comment:
Maybe worth to factorize with the other test? Not sure... just a suggestion to look into it.

Reply:
I made two tests to keep the assert on number of call odf the method `LTI.verify`